src/perform_step/linear_perform_step.jl

```diff
@@ -819,6 +819,7 @@ function perform_step!(integrator, cache::CayleyEulerConstantCache, repeat_step
         A = f.f
     end
 
+    # TODO: this is not in place, think about this
     L = update_coefficients(A, uprev, p, t)
```

Comment:
@ChrisRackauckas are there more uses of OOP `update_coefficients` in OrdinaryDiffEq outside of this line? This was the only one I could find. And is this line intentionally out-of-place?

I'm wondering because I want to know whether full-fledged for OOP `update_coefficients` in SciMLOperators is a blocker to integration into OrdinaryDiffEq or not. (It's something that should be done soon regardless, of course.)

Reply:
@ChrisRackauckas ping

Reply:
@gaurav-arya if this is the only OOP call, can we replace it with IIP and see if anything breaks?

Reply:
[resolved]